Update Readme.md (#826)
grosser/parallel_tests: Readme.md

    gem install parallel_tests
    # go to your project dir
    parallel_test
    parallel_rspec
    parallel_cucumber
    parallel_spinach

>  - use `ENV['TEST_ENV_NUMBER']` inside your tests to select separate db/memcache/etc. _(Remember to expose this to docker if using compose!)_

Reviewer: ```suggestion
 - use `ENV['TEST_ENV_NUMBER']` inside your tests to select separate db/memcache/etc. (docker compose: expose it)
```

>  - 

Reviewer: remove extra `-` ?

 - Only run a subset of files / folders:
 
    `parallel_test test/bar test/baz/foo_text.rb`

>    

Reviewer: remove space

 - Pass test-options and files via `--`:
 
    `parallel_rspec -- -t acceptance -f progress -- spec/foo_spec.rb spec/acceptance`
    

>  - Pass in any custom cucumber runtime option, by using the -o flag. **NB: You must wrap everything in quotes**:

Reviewer: ```suggestion
 - Pass in test options, by using the -o flag (wrap everything in quotes):
```
